nais/dev-gcp/nais.yaml

```diff
@@ -34,6 +34,25 @@ spec:
     requests:
       cpu: "1"
       memory: 2048Mi
+  gcp:
+    sqlInstances:
+      - name: brukernotifikasjon-cache
+        type: POSTGRES_11
+        tier: db-f1-micro
+        diskType: SSD
+        highAvailability: false
+        diskSize: 10
+        diskAutoresize: false
+        autoBackupTime: "03:00"
+        databases:
+          - name: brukernotifikasjon-cache
+            envVarPrefix: DB
+            users:
+            - name: eventhandler
```

Comment:
Stemmer det at users her er en liste? Da hadde det kanskje gått an å lagt til flere brukere? For å begrense litt hvor mange endringer vi må gjøre med en gang, rundt hvilke apper som har tilgang til hva.